src/picongpu/include/particles/ParticlesFunctors.hpp

```diff
@@ -155,29 +155,66 @@ struct CallUpdate
     }
 };
 
-/* Tests if species can be ionized and calls the function to do that */
+/** \struct CallIonization
```

Comment:
Yeah it does realize that the documentation is for the following [class](http://www.stack.nl/~dimitri/doxygen/manual/commands.html#cmdclass)
Actually it is:
```
/** \struct ClassName
 *
 * \brief shorter description (one-liner maybe)   
 *   
 * longer description ... blabla
 *  
 * \tparam T_SpeciesName [... description ...]
 */
```

I just skipped the longer one